lib/internal/assert/utils.js

```diff
@@ -199,7 +198,7 @@ function getErrMessage(message, fn) {
     // errors are handled faster.
     if (errorStackTraceLimitIsWritable) Error.stackTraceLimit = 0;
 
-    if (filename) {
```

Comment:
There is `if (filename) { ... } else { return ... }` structure above, and this check is unnecessary.